Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2201,6 +2201,25 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
         })
     }
 
+    @inlinable
+    func _vjpMean(alongAxes axes: [Int]) -> (Tensor, (Tensor) -> Tensor) {
+        let value = mean(alongAxes: axes)
+        let myShape = shape
```

Comment:
Is the temporary binding `let myShape = shape` necessary?
If so, could you please add a comment explaining why it's needed? Future readers of the code then won't be tempted to remove the line.